plugins/modules/docker_secret.py

```diff
@@ -99,16 +96,15 @@
 
 author:
   - Chris Houseknecht (@chouseknecht)
-'''
-
-EXAMPLES = '''
+"""
 
+EXAMPLES = r"""
 - name: Create secret foo (from a file on the control machine)
   community.docker.docker_secret:
     name: foo
-    # If the file is JSON or binary, Ansible might modify it (because
-    # it is first decoded and later re-encoded). Base64-encoding the
-    # file directly after reading it prevents this to happen.
+  # If the file is JSON or binary, Ansible might modify it (because
+  # it is first decoded and later re-encoded). Base64-encoding the
+  # file directly after reading it prevents this to happen.
```

Comment:
```suggestion
    # If the file is JSON or binary, Ansible might modify it (because
    # it is first decoded and later re-encoded). Base64-encoding the
    # file directly after reading it prevents this to happen.
```